### PR TITLE
Add `dart_wot` to list of WoT Implementations

### PR DESCRIPTION
**Pull request by JKRhb**
As briefly discussed in today's Scripting Call, this PR adds [`dart_wot`](https://pub.dev/packages/dart_wot) (a Scriping API implementation written in Dart) to the list of implementations on the WoT website.

### CODE REVIEW

**Review by danielpeintner — CHANGES_REQUESTED** [review]
for consistency reasons I would argue we should have bracket ("(" and ")") around the description since all the other items do have that.

File: docs/developers/index.html [hunk 1]
```diff
@@ -80,6 +80,9 @@ <h3>WoT Implementations</h3>
 		<li>
 			<a href="https://www.evosoft.com/en/digitalization-offering/saywot/" target="_blank">sayWoT!</a> (for web and cloud developers)
 		</li>
+		<li>
+			<a href="https://pub.dev/packages/dart_wot" target="_blank">dart_wot</a> W3C Web of Things implementation written in Dart with support for HTTP and CoAP.
```
[the remark this answers is not in the document]
Comment (danielpeintner) [reply docs/developers/index.html:84]: 👍

**Comment by egekorkan** [comment]
Any idea why netlify says "Failed during stage 'checking build content for changes': Canceled build due to no content change". Otherwise the change is ok for me

**Comment by JKRhb** [comment]
> Any idea why netlify says "Failed during stage 'checking build content for changes': Canceled build due to no content change".

I think this is probably caused by the fact that I squashed the commits. Therefore it didn't detect any changes on the last run in comparison to the (now non-existent) commit before. Can you rerun the preview manually?

**Comment by egekorkan** [comment]
Same problem but https://deploy-preview-289--wot-marketing.netlify.app/developers/ shows the changes

LGTM :)

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**